src/Messages/OWSMessageSender.m

```diff
@@ -486,36 +487,7 @@ - (void)resendMessageFromKeyError:(TSInvalidIdentityKeySendingErrorMessage *)err
     //  2.) fail and create a new identical error message in the thread.
     [errorMessage remove];
 
-    if ([errorMessage.thread isKindOfClass:[TSContactThread class]]) {
```

Comment:
This is a significant change. When re-sending group messages that failed to safety number changes, we used a special code path to avoid double-sending to other members of the group.  That code path was mostly redundant with our normal message sending code path but had some differences. Most significantly it ignored the message send operation queue.  I've consolidated along the one true code path, for consistency and correctness.  We now avoid double-sending to other members of a group using another mechanism which also applies to message send retries.